### ISSUE TITLE
malachite 

### ISSUE THREAD

**Issue by arademaker** [globalwordnet/english-wordnet#534]
http://wn.mybluemix.net/synset?id=14680661-n

by the Wikipedia definition, malachite should have more "Substance meronym of"?

https://en.wikipedia.org/wiki/Malachite

**Comment by jmccrae** [comment]
Targetting what? There is already a relation to 'copper' as the substance that composes this mineral

**Comment by arademaker** [comment]
http://wn.mybluemix.net/synset?id=14798450-n at least?

**Comment by jmccrae** [comment]
Seems like it should be a hypernym link not a meronym link?

**Comment by arademaker** [comment]
From the sentence wikipedia article first sentence 

> Malachite is a copper carbonate hydroxide mineral

I take Malachite as a kind of mineral containing copper (Cu2), carbonate (CO3), and hydroxide (OH)2. Note that carbonate is a mineral containing carbonate ion. Hydroxide is https://en.wikipedia.org/wiki/Hydroxide and it is maybe duplicated in PWN http://wn.mybluemix.net/search?search_field=word_en&term=hydroxide.

**Comment by arademaker** [comment]
Maybe @ alexandretessarollo can add something here.

**Comment by jmccrae** [comment]
Yes, but the synset for 'carbonate' refers to any salt (compound) containing CO3, while the synset for 'copper' refers to the element by itself. The hypernym link would be consistent with the other modelling, e.g., compare 'lithane' and 'bauxite'.

**Comment by alexandretessarollo** [comment]
I checked GBA Thesaurus and got this:
[http://resource.geolba.ac.at/mineral/427](http://resource.geolba.ac.at/mineral/427)
Chemical Formula: Cu2(CO3)(OH)2; Major elements: Cu; Minor and trace elements: As, Sb, Ni, Ca, Fe, Mg; Group: Carbonates. Label according to www.ima-mineralogy.org/Minlist; www.mindat.org; www.handbookofmineralogy.org (IMA 2014)

It gave other references with extra information as well, such as [https://www.mindat.org/min-2550.html](https://www.mindat.org/min-2550.html) (great some, tons of information). And according to [this reference](https://www.gemdat.org/gem-2550.html), it can be an ore of copper. One final reference is [http://www.handbookofmineralogy.org/pdfs/Malachite.pdf](http://www.handbookofmineralogy.org/pdfs/Malachite.pdf).


So, to sum up: it is a mineral; is can be an ore for copper (Cu); it is a carbonate; and it should contain more meronyms. Whether to point the meronym to C, O, H or to CO3, OH (or even both options) is a matter os design decision..

**Comment by jmccrae** [comment]
Closed by #569
